docker/build-tools/Dockerfile

```diff
@@ -99,10 +99,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	libc-dev \
 	make \
 	pkg-config \
-    wget \
-    && rm -rf /var/lib/apt/lists/*
+    wget
 
-# Not in a sharing mood
+# Clean up stuff we don't need in the final image
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
why not in the same `&&` chain as before? I thought that was a best practice - I am not a docker expert though

Reply:
Agree it should be with the other chain. I think this saved me a few percent in size when I added it.

Reply:
It is in an intermediate build so might not impact the final size by operating it out. Might only affect the layers.

Reply:
Indeed, the layers above the line  that says  "FROM scratch" are irrelevant. Below  that line it matters, and that's where I  use the clumsy && sequences.

Reply:
Oh nice, didn't realize that

Reply:
this is correct.  We want single liners - unless we are talking about the final rendered image.  Even the final rendered image doesn't matter much.  Layer reduction used to be super important with Docker because multiple layers made docker crash.  Now the whole planet thinks this is a best practice, however, it is a poor practice given the current implementation of docker.

Docker will download multiple layers at the same itme (this used to not be the case), so its actually more performant to have multiple layers.